docs/how-to-generate-diagrams.md

```diff
@@ -0,0 +1,9 @@
+#How to generate diagrams
+
+Proxy diagrams should be generated by preference with Plantuml.
+Plantuml is a tool that allows to generate diagrams in different image formats
```

Comment:
I'd consider shortening this to, "PlantUML is a tool that generates UML in different image formats from a textual description".